.github/workflows/build-check-install.yaml

```diff
@@ -167,6 +167,25 @@ jobs:
           name: ${{ matrix.config.tag }}-results
           path: ${{ github.event.repository.name }}.Rcheck/00check.log
 
+      - name: Publish Unit Test Results
+        uses: EnricoMi/publish-unit-test-result-action@v1
+        if: failure()
+        with:
+          files: "${{ github.event.repository.name }}.Rcheck/tests/testthat/junit-result.xml"
+
+      - name: Convert JUnit XML Test Results to Markdown
+        run: |
+          wget -q https://github.com/dinakar29/junit2md/releases/download/v0.1.0/junit2md_0.1.0_linux-amd64 -O /tmp/junit2md
```

Comment:
I have no clue what's going on here, but this reference to a particular user is suspicious to me. Can someone confirm it's alright?

Reply:
I hope so. It's me :)